tools_init/install_profile.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+. _core.sh
+
+debug "SONAR_TOKEN : $SONAR_TOKEN"
+debug "SONAR_URL : $SONAR_URL"
+debug "RULES_KEYS : $RULES_KEYS"
+debug "TAG_ECOCONCEPTION : $TAG_ECOCONCEPTION"
+debug "PROFILE_ECOCONCEPTION : $PROFILE_ECOCONCEPTION"
+debug "PROFILES_LANGUAGE_KEYS : $PROFILES_LANGUAGE_KEYS"
+
+# check SonarQube API connection
+check_sonarapi
+
+declare -i nb_profile_update=0
+
+# loop on each profile to create a fork SonarWay with + New rules ecoconception
+for language in "${PROFILES_LANGUAGE_ARRAY[@]}"
+do
+  echo -e "\n*****  Processing profile language ${BLUE}$language${NC}  *****"
+
+  create_profile_sonarapi "$language" "$PROFILE_ECOCONCEPTION"
```

Comment:
I think adding some logs to steps will help the user. For example :
- `echo -e "Creating profile : $PROFILE_ECOCONCEPTION"` before `create_profile_sonarapi` call
- `echo -e "Changing parent profile"` before `change_parent_profile_sonarapi` call
- `echo -e "Activating ecocode rules with tag '$TAG_ECOCONCEPTION' for key_profile '$key_profile'"` before `activate_rules_ecocode_profile_sonarapi` call

I did it locally to understand my local execution to check.